tools-local/unix/acquire-tool.sh

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env bash
+
+# Downloads the specified tool.
+# Download URL is read from the .toolversions file.
+
```

Comment:
Added comments describing arguments.

---
In reply to: [107791233](https://github.com/ravimeda/corefx/pull/2#discussion_r107791233) [](ancestors = 107791233)